src/ic-xrc-types/src/lib.rs

```diff
@@ -19,13 +19,13 @@ pub struct Asset {
 }
 
 /// The type the user sends when requesting a rate.
+///
+/// For definitions of "base", "quoted", etc, refer to https://en.wikipedia.org/wiki/Currency_pair
 #[derive(CandidType, Clone, Debug, Deserialize)]
 pub struct GetExchangeRateRequest {
-    /// The asset to be used as the resulting asset. For example, using
-    /// ICP/USD, ICP would be the base asset.
+    /// The base asset.
     pub base_asset: Asset,
-    /// The asset to be used as the starting asset. For example, using
-    /// ICP/USD, USD would be the quote asset.
+    /// The quoted asset.
```

Comment:
While not explicitly mentioned in the currency-pair wiki page, it is usually called "quote asset" (and not "quoted asset"). Google Trends agrees with this.   :slightly_smiling_face:

Reply:
Done.